webrtc.html

```diff
@@ -4071,7 +4071,11 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 
           <dd>
             <p>Returns the certificate chain in use by the remote side, with each certificate
-            encoded in binary Distinguished Encoding Rules (DER) [[!X690]].</p>
+            encoded in binary Distinguished Encoding Rules (DER) [[!X690]]. If the remote peer offers
+            multiple certificates, then <code>getRemoteCertificates()</code> will return null until
```

Comment:
`null` or the empty list?  Returning `null` would require that the return value is nullable, which it isn't.